reader/bin/new-migration.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+const fs = require('fs');
+const path = require('path');
+
+function createMigration(name) {
+  const timestamp = Date.now();
```

Comment:
could this be configurable also?

Reply:
we need the timestamp, to enforce order. The name is appended though